youtube_dl/extractor/picosong.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import urljoin, parse_duration, parse_filesize
+
+
+class PicosongIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?picosong\.com/(?P<id>[0-9A-Za-z]{4})'
+    _TESTS = [{
+        'url': 'http://picosong.com/7uer/',
+        'md5': '7f2ee011ba3ca2ac180a35deb10feabf',
+        'info_dict': {
+            'id': '7uer',
+            'ext': 'mp3',
+            'artist': 'Xarlable',
+            'title': 'Rock My Forum',
+            'album': 'Epic',
+            'genre': 'Howto & Style',
+            'duration': 187.0,
+            'filesize_approx': 4700000,
+            'asr': 44100,
+            'abr': 210,
+        },
+    }, {
+        'url': 'http://picosong.com/i3gs/',
+        'md5': 'd4ec5d64d97e28b45d2710c096c29f70',
+        'info_dict': {
+            'id': 'i3gs',
+            'ext': 'mp3',
+            'title': 'Rick.mp3',
+            'duration': 26.0,
+            'filesize_approx': 1020000,
+            'asr': 48000,
+            'abr': 320,
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        page = self._download_webpage(url, video_id)
+        download_path = self._search_regex(
+            r'value="/static/swf/picoplayer\.swf\?file=([^"]+)"',
+            page, 'url')
+        download_url = urljoin(url, download_path)
+
+        def extract_info_td(key, fatal=True):
```

Comment:
It's more rational to use `False` as default since it's used more often in this code.